src/components/comment/index.js

```diff
@@ -1,8 +1,36 @@
+import "./style.css"
+import ProfileCircle from "../profileCircle"
+import EditPostModal from "../editPostModal"
+import useModal from "../../hooks/useModal"
+import { useEffect } from "react"
+
+
+
 const Comment = ({ name, content }) => {
+    //fetch comments from api once it exists. talk to Server team
+    const { openModal, setModal } = useModal()
+    // const [comment, setComment] = useEffect([])
```

Comment:
Rather than comment lines of code out, just remove them. Git exists to keep a history of previous code so it's always possible to bring stuff back in if needed :)